RELEASING.md

```diff
@@ -0,0 +1,187 @@
+# How to Create a Release of OpenCensus Proto (for Maintainers Only)
+
+## Build Environments
+
+We re-generate gen-go files and deploy jars to Maven Central under the following systems:
+
+-   Ubuntu 14.04
+
+Other systems may also work, but we haven't verified them.
+
+## Prerequisites
```

Comment:
I think this belongs to the Release Java Jar section

Reply:
Correct, moved to "Release Java Jar" as a sub-title.